### DISCUSSION TITLE
Samples / documentation highlighting native embedding

### DISCUSSION THREAD

**Discussion by cosminstirbu** [dotnet/maui#6613] [General]
Hello,

Can you please point us to any samples / documentation highlighting how we can embed .NET MAUI in a native .NET iOS / Android application?

Thank you,
Cosmin

**Comment by rodrigojuarez** [comment]
Have you ever found any documents or samples of this?

**Reply by cosminstirbu** [reply]
There's a section in the official documentation related to this - https://learn.microsoft.com/en-us/dotnet/maui/platform-integration/native-embedding

I didn't test it myself though.
